Add contact form

arnoldnekemiah opened the pull request:
- Added a contact form
- Styled the contact form
- Added footer and styled it.
- Solved style-linter errors

Commit:
Added new branch

Commit:
created a new branch

Innocent259 committed:
Create contact form section

Commit:
styled the form

Commit:
Added more styling and solved style-linter errors

Commit:
Added more styling and solved style-linter errors

Tshepo45 committed:
Modified .Html File

-Added Value attribute to text input

osamaashraf6 commented on index.html:
```diff
@@ -233,6 +233,58 @@ <h2 class="footer-heading">🟠 Skills</h2>
+      so if you'd like to chat please 
+      get in touch.
+    </h2>
+    <form class="form" action="https://formspree.io/html/" method="post">
```
- [x] You did well 😊 at adding the form, but I see you forgot to add the Id for the form spree to make the users when write info reach you via the form spree, so please try to fix that for the requirements 😃[Click here ](https://youtu.be/sVPlb5C3oQ8) to watch this video on how to use the Form Spree💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| My form | Your form| 
| --- | --- |
| ![Recording 2023-02-28 at 22 00 13](https://user-images.githubusercontent.com/83360636/221977669-0af282ad-3aa6-4913-a9b4-15b93b61dc30.gif)  |  ![Recording 2023-02-28 at 22 02 08](https://user-images.githubusercontent.com/83360636/221965709-68d66e16-b1b5-46f7-a3ca-b280c79aca01.gif)   |

osamaashraf6 commented on index.html:
```diff
@@ -233,6 +233,58 @@ <h2 class="footer-heading">🟠 Skills</h2>
+      class="input-1"
+      type="text" value="Arnold"
+      maxlength="30"
+      placeholder="Full name"
+      required
+      >
+      <hr class="line-z">
+      </div>
+      <div class="input-username">
+        <input 
+      class="input-1"
+      type="email"
+      placeholder="Email address"
+      required
+      >
+      <hr class="line-z">
+      </div>
+      
+      <textarea class="textarea" maxlength="500">Enter text here</textarea>
```
- [ ] You did well 😊👌at adding the validation for the other two inputs, but I see that you didn't add the required for the textarea to make the user unable to submit an empty textarea, so please try to add the required for the textarea also 😃

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| My form | Your form| 
| --- | --- |
|  ![Recording 2023-02-28 at 21 56 35](https://user-images.githubusercontent.com/83360636/221964643-6bb51ad5-6d1a-4619-be92-1cf53b5ebd02.gif)   |    ![Recording 2023-02-28 at 21 54 57](https://user-images.githubusercontent.com/83360636/221964310-bb39cbf0-5587-47bf-bc47-9aacc0f27189.gif)  |

osamaashraf6 replied:
- [ ]  Kindly 😊 try to remove the content of the textarea because you make it static and the user has to remove it to write his info, so please try to make it placeholder like the Figma Design 😃

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| My form | Your form| 
| --- | --- |
|  ![Recording 2023-02-28 at 22 11 10](https://user-images.githubusercontent.com/83360636/221967518-c31167ea-7cc7-4858-bdae-c7c61f788b05.gif)  |    ![Recording 2023-02-28 at 22 11 57](https://user-images.githubusercontent.com/83360636/221967688-47d10c58-6f44-4419-88e4-5c1ea529875f.gif)  |

osamaashraf6 replied:
- [x] Kindly 😃 try to stick to the Figma Design, so please here I see that the textarae is outside of the align with from content, so please try to correct that to align it with the form 😊
- [x] Kindly try to fix the border of the elements in the form to be like the Figma design 😊as you see the border in yours is bolder than the Figma Design 💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/221972428-970dc465-0edb-42c0-b32b-8d7963a8c103.png)   |    ![image](https://user-images.githubusercontent.com/83360636/221977956-728e4c7d-ade7-45fa-90ac-b69bb0eb5d3f.png)  |

osamaashraf6 commented on index.html:
```diff
@@ -233,6 +233,58 @@ <h2 class="footer-heading">🟠 Skills</h2>
+    <form class="form" action="https://formspree.io/html/" method="post">
+      <div class="input-username">
+        <input
+      class="input-1"
+      type="text" value="Arnold"
+      maxlength="30"
```
- [ ] Kindly 😊 try to remove the value of the input because you make it static and the user has to remove it to write his info, so please try to make it placeholder like the Figma Design 😃

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| My form | Your form| 
| --- | --- |
|  ![Recording 2023-02-28 at 22 08 09](https://user-images.githubusercontent.com/83360636/221966917-3d89102e-00fc-46e3-973b-1ff53c21dd4a.gif)  |    ![Recording 2023-02-28 at 22 06 41](https://user-images.githubusercontent.com/83360636/221966629-2b78704d-d05c-4b83-9389-895c22c9ea1c.gif) |

osamaashraf6 commented on index.html:
```diff
@@ -233,6 +233,58 @@ <h2 class="footer-heading">🟠 Skills</h2>
       </div>
     </div>
   </section>
+  <section class="contact">
+      <h2 class="conclusion-txt">
+      I'm always interested in 
+      hearing about new projects, 
+      so if you'd like to chat please 
+      get in touch.
+    </h2>
```
- [x] Kindly 😃 try to use the correct font size for the heading to be like the Figma Design as I see you put the size of text here as 30 px, but in the Figma is 32px, so please try to fix that 😊
- [x] Also please try to use the correct font weight to be like the Figma Design 😊
- [x] Please try to decrease the margin-top of the heading here to be like the Figma design💯 

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/221970496-f24aa01a-e015-41b7-8404-483b044ae568.png)  |    ![image](https://user-images.githubusercontent.com/83360636/221970573-60a4efa7-1369-4559-bcb2-5eb265070446.png)  |

osamaashraf6 commented on index.html:
```diff
@@ -233,6 +233,58 @@ <h2 class="footer-heading">🟠 Skills</h2>
       </div>
     </div>
   </section>
+  <section class="contact">
+      <h2 class="conclusion-txt">
+      I'm always interested in 
```
- [ ] Kindly 💯 try to fix the scroll of the page because when trying to move the mouse on the page I see that there is scrolling in the page, so please try to fix that to be like the Figma design and to be great for the users 😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Your design| 
| --- |
|   ![Recording 2023-02-28 at 22 46 57](https://user-images.githubusercontent.com/83360636/221976172-ee776b20-916a-4b51-88c8-f24786fd5103.gif)   |

osamaashraf6 commented on index.html:
```diff
@@ -233,6 +233,58 @@ <h2 class="footer-heading">🟠 Skills</h2>
+      <button class="btn-submit" type="submit">Get in touch</button>
+    </form>
+  </section>
+  <footer class="footer-last">
+      <ul class="brands1">
+       <li class="icon">
+        <a href="#"><img src="./Images/github.png" alt="github"></a>
+       </li>
+       <li class="icon">
+        <a href="#"><img src="./Images/linkedIn.png" alt="linkedIn"></a>
+       </li>
+       <li class="icon">
+        <a href="#"><img src="./Images/peace.png" alt="Angellist"></a>
+       </li>
+       <li class="icon">
+        <a href="#"><img src="./Images/twitter.png" alt="twitter"></a>
+       </li>
+       <li class="icon">
+        <a href="#"><img src="./Images/m.png" alt="m"></a>
+       </li>
+      </ul>  
```
- [x] You did well 😃here at adding the icons, but please try to stick to the design as I see here that the size of the icons is bigger than the Figma design, so please try to fix that to be like the Figma Design 😊
- [x] Also please try to add the hr at the top of the icons to be like the Figma design 💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|![image](https://user-images.githubusercontent.com/83360636/221977161-1df23ab7-2324-4909-9e8b-4333f0c017e6.png)   |    ![image](https://user-images.githubusercontent.com/83360636/221977080-b42f7d77-683d-4e1d-a64e-b9335a860434.png) |

osamaashraf6 reviewed (changes requested):
Hi @arnoldnekemiah 👋, 

Good job so far! 👏🥳
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

![you-are-near](https://media.giphy.com/media/SSR17K6yZWvGCYrfUh/giphy.gif)

## Highlights ✔️


- [x] Nice design 
- [x] No linter errors. ✔️
- [x] Good usage in correcting GitHub flow.✔️





## Required Changes ♻️
- Kindly 😊 try to add the correct link for the live demo as when I click on it it is not open the live demo on the page, so try o put the correct one to let the visitors see your project
- I see that you put your contact info only Also kindly try to add the information contacts of your partner at the readme as this project pair programming, so kindly try to update it with your info contacts and your partner to let the user see you 😃
- check the comments under the review.

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Corrected style.css and html file as suggested by the reviewer

Commit:
Solved style-linter errors

arnoldnekemiah committed:
Update Readme.md File

Added contacts of collaborators
Added link to project live demo

arnoldnekemiah commented:
Attended to all comments by the preview

- Updated Readme.md file with contact information for my collaborators
- Added link to project live demo
- Attended to the HTML and CSS issues as suggested by the reviewer.
- Solved Style-linter errors

AdnanOlarmmi commented on index.html:
```diff
@@ -233,6 +233,47 @@ <h2 class="footer-heading">🟠 Skills</h2>
   </section>
+  <section class="contact">
+      <h2 class="conclusion-txt">
+      I'm always interested in 
+      hearing about new projects, 
+      so if you'd like to chat please 
+      get in touch.
```
- [ ] Like the previous spoke about this part, kindly align this particular item to the center like it is in the Figma design, to enable a pixel-perfect design 😀.

AdnanOlarmmi reviewed (changes requested):
Hi @arnoldnekemiah, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

![You're Almost there, funny one eyed alien](https://user-images.githubusercontent.com/41163571/217957948-6d8d9f33-125c-4724-954f-330eed60533c.gif)

## Highlights
- Professional Readme.
- Almost done.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Aligned h2 text to center as suggested by reviewer

arnoldnekemiah commented:
Aligned h2 text to center as suggested by previous reviewer

leonardodiasb reviewed (approved):
# Status: ✔️ Project Approved ✔️ 

Hello @arnoldnekemiah 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - Nice design
 - No linter errors. ✔️
 - Good usage in correcting GitHub flow.✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
